src/api.ts

```diff
@@ -23,6 +23,7 @@ function get<T>(url: string): Promise<T> {
     .then(res => res.json());
 }
 
-export const getTodos = () => get<Todo[]>('/todos');
+export const getTodos: () => Promise<Todo[]> = () => get<Todo[]>('/todos');
```

Comment:
Ensure that the `/todos` endpoint returns data in the expected format for `Todo[]`. This is crucial for the application to function correctly.